client/layout/masterbar/woo-core-profiler.tsx

```diff
@@ -3,6 +3,9 @@ import { Button } from '@wordpress/components';
 import { getQueryArg } from '@wordpress/url';
 import { localize } from 'i18n-calypso';
 import { Fragment } from 'react';
+// @ts-expect-error Missing definition
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
+import validUrl from 'valid-url';
```

Comment:
It should pretty easy to type this one.

Create an `index.d.ts` in the same folder with the following contents:

```ts
declare module 'valid-url' {
	export function isWebUrl( url: string ): boolean;
}
```